python/ballista/tests/test_context.py

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from pyballista import SessionContext
+from ballista import SessionContext
```

Comment:
there is no `SessionContext` anymore, is there ? 

```bash
python3 -m pytest
```

```
==================================================================================================== ERRORS ====================================================================================================
_______________________________________________________________________________ ERROR collecting ballista/tests/test_context.py ________________________________________________________________________________
ImportError while importing test module '/Users/marko/git/arrow-ballista/python/ballista/tests/test_context.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/Cellar/python@3.12/3.12.6/Frameworks/Python.framework/Versions/3.12/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ballista/tests/test_context.py:18: in <module>
    from ballista import SessionContext
E   ImportError: cannot import name 'SessionContext' from 'ballista' (/Users/marko/git/arrow-ballista/python/ballista/__init__.py)
=========================================================================================== short test summary info ============================================================================================
E
```

Reply:
updated

Reply:
tests passing